src/main/java/com/google/sps/servlets/JoinRoomUtil.java

```diff
@@ -0,0 +1,6 @@
+package com.google.sps.servlets;
+import com.google.gson.Gson;
+
+public class JoinRoomUtil {
```

Comment:
Make this class name more generic, like ServletUtil.

Also, add class level documentation